app/components/DR/styles/styles.js

```diff
@@ -0,0 +1,234 @@
+import { StyleSheet } from 'react-native';
```

Comment:
Could we use the styled per component or inside the components? 

It will be difficult to maintain this

Reply:
Implemented